src/views/Reports/InvestmentReport.jsx

```diff
@@ -29,8 +29,13 @@ const useStyles = makeStyles(theme => ({
     position: 'sticky',
     left: 0,
     background: '#dbdbdb',
-    top: '57px',
-    zIndex: 900
+    zIndex: 900,
+    [theme.breakpoints.down('md')]: {
```

Comment:
Why not 'lg'?
<img width="541" alt="image" src="https://user-images.githubusercontent.com/67354752/188647238-6b6603e9-fa3d-4e84-aa03-5ed6fb56884d.png">